discordrb.json

```diff
@@ -57,6 +57,18 @@
             "voiceReceive": "no",
             "voiceSend": "yes",
             "voiceStateUpdate": "yes"
+        },
+        "misc": {
+            "supportsUserLogin": "yes",
+            "supportsBotLogin": "yes",
+            "cachesTokens": "yes",
+            "apiDocs": "inline",
```

Comment:
`inline` documentation can also have external generation (or not). For example discord.py has inline documentation along with external but discord.net has inline documentation but no "official" external generation. 

This needs better selection.